doc/source/reference/routines.other.rst

```diff
@@ -35,3 +36,33 @@ NumPy version comparison
    :toctree: generated/
 
    lib.NumpyVersion
+
+NumPy header files location
+----------------------------
+.. autosummary::
+   :toctree: generated/
+   
+   get_include
```

Comment:
Should ```'get include'``` be under here [NumPy-specific functions](http://www.numpy.org/devdocs/reference/routines.help.html#reading-help)?

Reply:
Good to look for such things, but those are numpy specific *help* functions.

Reply:
got it, thanks!